sourcegraph/sourcegraph
# Code Insights: Add limited access mode UI and check for the creation and editing UI

## Context
Based on https://github.com/sourcegraph/sourcegraph/pull/32417

## Test plan
- Make sure that you can see all limited access mode UI and check when you don't have a license tag
- Make sure that all feature at the creation and editing UI work properly without any limitation when you have code insights license tag
